moped-editor/src/layouts/DashboardLayout/TopBar.js

```diff
@@ -72,7 +72,7 @@ const TopBar = ({ className, ...rest }) => {
     <AppBar
       sx={{ backgroundColor: "background.paper" }}
       elevation={2}
-      position="static" // Set static this component works with flex set in DashboardLayout
+      position="sticky"
```

Comment:
This fixed the bug shown in https://github.com/cityofaustin/atd-data-tech/issues/25683